include/openmp/Level.h

```diff
@@ -0,0 +1,47 @@
+#ifndef FAASM_LEVEL_H
+#define FAASM_LEVEL_H
+
+#include <mutex>
+#include <proto/faasm.pb.h>
+#include <util/barrier.h>
+#include <util/environment.h>
+
+namespace wasm {
+    namespace openmp {
+
+        // Global variables controlled by level master
+        class OMPLevel {
+        public:
+            const int depth = 0; // Number of nested OpenMP constructs, 0 for serial code
+            const int effective_depth = 0; // Number of parallel regions (> 1 thread) above this level
+            int max_active_level = 1; // Max number of effective parallel regions allowed from the top
```

Comment:
Will do this in a later PR since this class will need to change anyway. It's a bit confusing to be in the middle of Clang OpenMP which (righly ;)) uses snake case and Faasm, but I agree it clashes too much so I'll change this and also some in `openmp.cpp`